src/wasm.cc

```diff
@@ -141,14 +141,18 @@ void WasmBase::registerCallbacks() {
                                    exports::_fn>::convertFunctionWordToUint32);
   _REGISTER_PROXY(log);
 
+  _REGISTER_PROXY(get_configuration);
   _REGISTER_PROXY(get_status);
 
   _REGISTER_PROXY(set_property);
   _REGISTER_PROXY(get_property);
 
+  _REGISTER_PROXY(continue_request);
```

Comment:
Shouldn't these be conditional based on the version?

Reply:
See https://github.com/envoyproxy/envoy-wasm/pull/596#issuecomment-668195656:

> Note, that because we register/expose host functions before linking, we don't know the exported ABI version marker at that point, so we expose host functions for both 0.1.0 and 0.2.0 version of the ABI (since they are not conflicting), and the switching is only happening for the callbacks.